docs/gallery/example_term_set.yaml

```diff
@@ -0,0 +1,23 @@
+id: pynert/termset/species_example
+name: Species
```

Comment:
Can we also add a `version: 0.0.1` here to make sure all TermSets are versioned?

Reply:
Sure. This isn't in the format LinkML specifies in the tutorials, but I think it is a good addition.